pkg/controller/secret/secret_controller_test.go

```diff
@@ -188,7 +188,7 @@ func Test_addPDSecretToAlertManagerConfig(t *testing.T) {
 			"severity",
 		},
 		MatchRE: map[string]string{
-			"namespace": "^openshift$|openshift-.*|default$|kube$|kube-.*|logging$",
+			"namespace": "^kube-.*,^openshift-.*,^logging$,^default$,^openshift$,^ops-health-monitoring$,^ops-project-operation-check$,^management-infra$",
```

Comment:
Since this is set in two places and it's pretty complex I'd prefer seeing it as a constant, but given this isn't owned by this operator I'll let it slide.  Unit test failures will catch changes out of sync?

Reply:
Yeah, the unit test caught the mismatch this time. I could make it a constant and export it to the packages that need it.

Reply:
Do you want me to do that on this PR?

Reply:
Since you need some more eyes on it for a lgtm label, might as well.